src/components/search/type-section/type-section.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {Label} from '../../common/label/label.js';
+import {Button} from '../../common/button/button.js'
+
+const TypeSection = (props) => (
+  <div className={'search-type-section'}>
+    <Label content={'search by'} labelUC={true} labelWhite={true}/>
+    {
+      props.searchTypes.list.map((item) => (
+          <Button btnRed={item === props.searchTypes.active}
+                  btnGrey={item !== props.searchTypes.active}
+                  key = {item}
+                  id = {item}
+                  content = {item}
```

Comment:
Do we really need to have `item` as a property 3 times?

Reply:
reduced